test/benchmark/simulation.cpp

```diff
@@ -69,9 +69,9 @@ TEST_CASE("Benchmark simulators", "[benchmark]")
         return quicksim<sidb_cell_clk_lyt_siqad>(lyt, quicksim_params);
     };
 }
-
 //  Mac M1, Ventura 13.0, Apple clang version 14.0.0 (22.11.23)
 
+//    old:
```

Comment:
What do "old" and "new" mean? We've probably forgotten soon. Maybe add a bit more description.